app/grandchallenge/challenges/models.py

```diff
@@ -809,27 +761,33 @@ def add_participant(self, user):
         if user != get_anonymous_user():
             user.groups.add(self.participants_group)
             follow(
```

Comment:
I had to update the follows here already because otherwise all tests including the ChallengeFactory would fail.... Migration of the existing follows will be done in a separate PR.